code/modules/hydroponics/trays/tray.dm

```diff
@@ -124,6 +124,17 @@
 		)
 
 /obj/machinery/portable_atmospherics/hydroponics/AltClick()
+	if (istype(usr, /mob/living/carbon/alien/diona))//A diona alt+clicking feeds the plant
```

Comment:
nymphs aren't human.
Afaik, doing that would give the ability to gestalts and take it away from nymphs, which is the opposite of what this is supposed to do